SU-SOE/soe_basic
# D8CORE-3246 D8CORE-3487: fixing the external link icons

# READY FOR REVIEW

# Summary
- Fixing the external link and SOE link icons for Publication and News.

# Review By (Date)
- 2/12

# Urgency
- Med

# Steps to Test

1. Pull in the changes
2. Make sure you have publications and news articles that are on the site and have external links.
3. Verify both Publication and News cards have both kind of link icons (soe arrow and ext link arrow)
4. Lists of News articles and Publications do not have the Soe arrow. Please take a look at the examples in Jira.

# Affected Projects or Products
- SOE Theme
- News
- Publications

# Associated Issues and/or People
- D8CORE-3264
- D8CORE-3487

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
